src/amo/components/Addon/index.js

```diff
@@ -183,11 +183,13 @@ export class AddonBase extends React.Component {
 
     return (
       <div className="Addon-icon" key="Addon-icon-header">
-        <img
-          alt={label}
-          className="Addon-icon-image"
-          src={getAddonIconUrl(addon)}
-        />
+        <div className="Addon-img-wrapper">
```

Comment:
Could we use the parent `div` (`.Addon-icon`) or do we need another `div` here? If we have to keep this new `div`, can we name it `Addon-icon-wrapper`?

Reply:
I guess we will not be able to apply the height and width constraints in the parent div.

Yes sure, I am changing the name to `Addon-icon-wrapper` and pushing it in a minute.
Thanks for the review!  :)

Reply:
P.S - I believe having an image cropper in place would have been the best solution! according to me, but that's totally another issue. :)